### PR TITLE
TW-1542: make message highlight brighter

### PR DESCRIPTION
**Pull request by sherlockvn**
## Ticket:
#1542 

## Demo:

https://github.com/user-attachments/assets/0c1046e1-e394-4189-ac91-c8bd24fc7862

### CODE REVIEW

File: lib/pages/chat/chat_event_list.dart [hunk 1]
```diff
@@ -150,7 +149,7 @@ class ChatEventList extends StatelessWidget {
                     key: ValueKey(event.eventId),
                     index: index,
                     controller: controller.scrollController,
-                    highlightColor: LinagoraRefColors.material().primary[99],
+                    highlightColor: Theme.of(context).highlightColor,
```
Comment (nqhhdev) [inline lib/pages/chat/chat_event_list.dart:152]: IMO: - This is a bad idea.

- Why do you use this color?
- Need to confirm with designer, team?
- Compared to the old color, I find it even worse.

Comment (sherlockvn) [reply lib/pages/chat/chat_event_list.dart:152]: `Theme.of(context).highlightColor` is based on default color of material design 3, there is no design for this from beginning. And change to global color, later we only need to change in one place (ThemeData file).

**Review by KhaledNjim — APPROVED** [review]

**Comment by KhaledNjim** [comment]
Tested on Android and web. Highlighted messages look good to me.

File: lib/pages/chat/chat_event_list.dart (hunk 1 above)
Comment (Te-Z) [reply lib/pages/chat/chat_event_list.dart:152]: Tested on android and web LGTM.
Also discussed with Bob, looks good for now as this screen will evolve in near future

**Review by Te-Z — APPROVED** [review]

File: lib/pages/chat/events/message/message.dart [hunk 2]
```diff
@@ -258,6 +258,7 @@ class _MessageState extends State<Message> {
                 constraints: BoxConstraints(
                   maxWidth: ChatViewBodyStyle.chatScreenMaxWidth,
                 ),
+                padding: const EdgeInsets.symmetric(vertical: 2.0),
```
Comment (nqhhdev) [inline lib/pages/chat/events/message/message.dart:261]: Move it to `lib/pages/chat/events/message/message_style.dart`

**Comment by sherlockvn** [comment]
Updated based on new design:

image


https://github.com/user-attachments/assets/d7805d77-d67c-4dd5-a5d2-3ebdf9b9e8b0

**Comment by sherlockvn** [comment]
#1932 

https://github.com/user-attachments/assets/8d93ef81-d7a4-4da0-9521-95d61aa2e0ab

**Review by nqhhdev — APPROVED** [review]
Tested on Web browser and Mobile (iOS and Android). LGTM

**Pull request merged**